packages/salesforcedx-apex-debugger/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@salesforce/salesforcedx-apex-debugger",
+  "displayName": "Apex Debugger Adapter",
+  "description": "Implements VSCode debug protocol for Apex Debugger",
```

Comment:
@ruthemmanuelle Just to provide context, this will not show up in the marketplace. This is the separate standalone NPM module. That being said, if there are description conventions to follow, let's use that.